pom.xml

```diff
@@ -178,22 +178,20 @@
             <artifactId>google-cloud-secretmanager</artifactId>
         </dependency>
 
-        <dependency>
-            <groupId>com.azure</groupId>
-            <artifactId>azure-security-keyvault-secrets</artifactId>
-            <version>${azure.keyvault.version}</version>
-        </dependency>
 
         <dependency>
-            <groupId>com.azure</groupId>
-            <artifactId>azure-identity</artifactId>
-            <version>${azure.identity.version}</version>
+            <groupId>com.azure.spring</groupId>
+            <artifactId>spring-cloud-azure-dependencies</artifactId>
+            <version>${com.azure.spring.version}</version>
+            <type>pom</type>
+            <scope>import</scope>
         </dependency>
 
+
         <dependency>
-            <groupId>com.microsoft.azure</groupId>
-            <artifactId>azure-keyvault-secrets-spring-boot-starter</artifactId>
-            <version>${azure.keyvault.spring.version}</version>
+            <groupId>com.azure.spring</groupId>
+            <artifactId>spring-cloud-azure-starter-keyvault</artifactId>
+            <version>${com.azure.spring.version}</version>
```

Comment:
If using with \<dependencyManagement>, there is no need to add the \<version> here. Because the BOM manages the version.

Reply:
Thank you @saragluna ! fixed this one :)